app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt

```diff
@@ -202,8 +202,8 @@ class EditSectionActivity : BaseActivity() {
     }
 
     private fun doSave(token: String) {
-        var summaryText = if (sectionHeading.isNullOrEmpty() ||
-            StringUtil.addUnderscores(sectionHeading) == pageTitle.prefixedText) "/* top */" else "/* $sectionHeading */ "
+        var summaryText = if (sectionAnchor.isNullOrEmpty() || sectionAnchor == pageTitle.prefixedText) "/* top */"
```

Comment:
This comparison needs `addUnderscores(sectionAnchor)`

Reply:
The `sectionAnchor` has already contained underscores, should I still need to apply the `addUnderscores`?

Reply:
When I try to edit an article with spaces in the title, the `sectionAnchor` still has spaces, not underscores.  And in fact when I edit an article that has a title with HTML in it, such as today's featured article (_Grevillea juniperina_), the `sectionAnchor` has HTML tags in it.

Reply:
Got it. Thanks for checking. Done.